Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -400,30 +400,35 @@ extension FirefoxHomeViewController: UICollectionViewDelegateFlowLayout {
         case UICollectionView.elementKindSectionHeader:
-                let view = collectionView.dequeueReusableSupplementaryView(ofKind: UICollectionView.elementKindSectionHeader, withReuseIdentifier: "Header", for: indexPath) as! ASHeaderView
-                let title = Section(indexPath.section).title
-                switch Section(indexPath.section) {
-                case .pocket:
-                    view.title = title
-                    view.moreButton.isHidden = false
-                    view.moreButton.setTitle(Strings.PocketMoreStoriesText, for: .normal)
-                    view.moreButton.addTarget(self, action: #selector(showMorePocketStories), for: .touchUpInside)
-                    view.titleLabel.textColor = UIColor.Pocket.red
-                    view.titleLabel.accessibilityIdentifier = "pocketTitle"
-                    view.moreButton.setTitleColor(UIColor.Pocket.red, for: .normal)
-                    return view
-                case .topSites:
-                    view.title = title
-                    view.titleLabel.accessibilityIdentifier = "topSitesTitle"
-                    view.moreButton.isHidden = true
-                    view.titleLabel.textColor = .black
-                    return view
-                case .libraryShortcuts:
-                    view.title = title
-                    view.titleLabel.textColor = .black
-                    view.titleLabel.accessibilityIdentifier = "libraryTitle"
-                    return view
+            let view = collectionView.dequeueReusableSupplementaryView(ofKind: UICollectionView.elementKindSectionHeader, withReuseIdentifier: "Header", for: indexPath) as! ASHeaderView
+            let title = Section(indexPath.section).title
+            switch Section(indexPath.section) {
+            case .pocket:
+                view.title = title
+                view.moreButton.isHidden = false
+                view.moreButton.setTitle(Strings.PocketMoreStoriesText, for: .normal)
+                view.moreButton.addTarget(self, action: #selector(showMorePocketStories), for: .touchUpInside)
+                view.titleLabel.textColor = UIColor.Pocket.red
+                view.titleLabel.accessibilityIdentifier = "pocketTitle"
+                view.moreButton.setTitleColor(UIColor.Pocket.red, for: .normal)
+                return view
+            case .topSites:
+                view.title = title
+                view.titleLabel.accessibilityIdentifier = "topSitesTitle"
+                view.moreButton.isHidden = true
+            case .libraryShortcuts:
+                view.title = title
+                view.titleLabel.accessibilityIdentifier = "libraryTitle"
+            }
+
+            let theme = BuiltinThemeName(rawValue: ThemeManager.instance.current.name) ?? .normal
+            if theme == .dark {
+                view.titleLabel.textColor = .white
+            } else {
+                view.titleLabel.textColor = .black
             }
```

Comment:
@electricRGB Wondering if we should put this in ASHeaderView?

Reply:
Fixed. :D

Reply:
yay!